src/main/resources/assets/owo/lang/es_ar.json

```diff
@@ -0,0 +1,55 @@
+{
+  "text.owo.itemGroup.tab_template": [
+    { "index": 0 },
+    { "text": " > ", "color": "gray" },
+    { "index": 1, "color": "dark_gray" }
+  ],
+  "text.owo.itemGroup.select_hint": {"text": "Shift + clic para seleccionar varios", "color": "gray"},
+  "text.owo.configure_hot_reload.title": "Configurar recarga en caliente",
```

Comment:
this translation is quite literal and I don't think it’s very clear, I think it could be `Configurar recarga instantánea` or leave `hot reload` in English

the other thing I would add is that the JSON file should probably be cloned to the other variants of Spanish since there’s nothing here that isn’t neutral Spanish (es_ar, es_cl, es_ec, es_es, es_mx, es_uy and es_ve)